k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -318,13 +307,8 @@ public void onCryptoModeChanged(CryptoMode cryptoMode) {
 
     private QuoteStyle mQuoteStyle;
 
-    private boolean mDraftNeedsSaving = false;
-    private boolean mPreventDraftSaving = false;
-
-    /**
-     * If this is {@code true} we don't save the message as a draft in {@link #onPause()}.
-     */
-    private boolean mIgnoreOnPause = false;
+    private boolean draftNeedsSaving = false;
```

Comment:
just for consistency: these are not preceded by 'm', the other private variables are. is there a reason?